### ISSUE TITLE
Logger can't be configured using json file.

### ISSUE THREAD

**Issue by gregoryb** [fluent/fluent-logger-node#44]
The logger seems not be configurable using a json file.
```
{
  "appenders": [
    {
        "type": "fluent-logger",
        "host": "localhost",
        "port": 24224,
        "timeout": 3.0,
        "reconnectInterval": 600000
    }
  ]
}
```

The module does not expose an "appender" function like mentionned in the doc. This function is inside "support" and options are not binded correctly.

From log4js:

```
module.exports.appenders[appender] = appenderModule.appender.bind(appenderModule);
```

**Comment by okkez** [comment]
Reproduced.
I will fix this issue in next week.

```js
var log4js = require('log4js');
log4js.configure('config.json');

var logger = log4js.getLogger();
logger.info("test1");
logger.info("test2");
logger.info("test3");
logger.info("test4");
```

with config file in https://github.com/fluent/fluent-logger-node/issues/44#issue-129081571

Got error:

```text
/home/kenji/ruby/td-family/spike/logger-node/node_modules/log4js/lib/log4js.js:291
      throw new Error(
      ^

Error: Problem reading log4js config { appenders:
   [ { type: 'fluent-logger',
       host: 'localhost',
       port: 24224,
       timeout: 3,
       reconnectInterval: 600000 } ] }. Error was "Cannot read property 'bind' of undefined" (TypeError: Cannot read property 'bind' of undefined
    at loadAppender (/home/kenji/ruby/td-family/spike/logger-node/node_modules/log4js/lib/log4js.js:411:63)
    at /home/kenji/ruby/td-family/spike/logger-node/node_modules/log4js/lib/log4js.js:226:7
    at Array.forEach (native)
    at configureAppenders (/home/kenji/ruby/td-family/spike/logger-node/node_modules/log4js/lib/log4js.js:225:18)
    at configureOnceOff (/home/kenji/ruby/td-family/spike/logger-node/node_modules/log4js/lib/log4js.js:283:7)
    at Object.configure (/home/kenji/ruby/td-family/spike/logger-node/node_modules/log4js/lib/log4js.js:346:3)
    at Object.<anonymous> (/home/kenji/ruby/td-family/spike/logger-node/issue44.js:4:8)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32))
    at configureOnceOff (/home/kenji/ruby/td-family/spike/logger-node/node_modules/log4js/lib/log4js.js:291:13)
    at Object.configure (/home/kenji/ruby/td-family/spike/logger-node/node_modules/log4js/lib/log4js.js:346:3)
    at Object.<anonymous> (/home/kenji/ruby/td-family/spike/logger-node/issue44.js:4:8)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:442:10)
    at startup (node.js:136:18)
    at node.js:966:3
```

**Comment by okkez** [comment]
Released v1.1.0
